translations/ui-circulation/en.json

```diff
@@ -275,5 +276,16 @@
   "settings.noticePolicy.loanNotices.renewed": "Renew",
   "settings.noticePolicy.loanNotices.checkIn": "Check in",
   "settings.noticePolicy.loanNotices.checkOut": "Check out",
-  "settings.noticePolicy.notices.andEvery": "and every"
+  "settings.noticePolicy.notices.andEvery": "and every",
+
+  "settings.loanHistory.validate.intervalValue": "Please enter a whole number greater than zero to continue",
```

Comment:
It looks like this file contains translations "Please enter a whole number greater than zero to continue", "Please select to continue", "Select interval", "Save" and these changes will add some duplications. @VictorSoroka1  what do you think about it?

Reply:
Agree.

Reply:
The first three occur twice, but the "Save" - 5 times. Maybe it will be additional story points for refactoring? @VictorSoroka1 what do you think about it?

Reply:
Yeah, agree. Please add just key:

`"save": "Save"`